### DISCUSSION TITLE
How to disable screen rotation on reTerminal | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1080] [Q&A]
# How to disable screen rotation on reTerminal | Seeed Studio Wiki

reTerminal-FAQ

https://wiki.seeedstudio.com/disable_screen_rotation_on_reTerminal/

**Comment by CALPARDO** [comment]
What if I want a static portrait?

**Reply by CALPARDO** [reply]
Anyway, after blacklisting the accelerator like above you can change the screen orientation through Raspberry icon on top left > Preferences> Screen Configuration > Configure > Screens > DSI-1 > Orientation > Right and reboot it works!
